test/e2e/e2e_test.go

```diff
@@ -81,10 +81,11 @@ var _ = Describe("Workload cluster creation", func() {
 			}, result)
 			cluster = result.Cluster
 			targetCluster = bootstrapClusterProxy.GetWorkloadCluster(ctx, namespace, clusterName)
-			test_remediation()
+			reMediation()
```

Comment:
Can we ignore remediation

Reply:
Is it something that we don't want in release-0.5?

Reply:
no just to test cluster delete quickly but if you think that tests will pass then should be kept in the test